docker/Dockerfile

```diff
@@ -108,30 +126,41 @@ RUN ["/bin/bash", "-c", "if [ -n $INSTALL_CERTS ]; then \
 RUN wget https://github.com/glattercj/vmdb2/releases/download/v1.1/vmdb2 -O /usr/bin/vmdb2 \
   && chmod +x /usr/bin/vmdb2
 
-# do this before installing phenix-apps so minimega package is latest version
-COPY --from=minimega /opt/minimega/lib /tmp/minimega
-RUN  python3 -m pip install /tmp/minimega
-
-ARG APPS_REPO=sandialabs/sceptre-phenix-apps
-ARG APPS_BRANCH=main
-
-RUN python3 -m pip install \
-  --trusted-host pypi.org \
-  --trusted-host files.pythonhosted.org \
-  "git+https://github.com/${APPS_REPO}.git@${APPS_BRANCH}#egg=phenix-apps&subdirectory=src/python"
-
 # needed to build Kali images with phenix
 RUN wget -O kali.deb https://http.kali.org/kali/pool/main/k/kali-archive-keyring/kali-archive-keyring_2022.1_all.deb \
-	&& dpkg -i kali.deb && rm kali.deb
+	&& dpkg -i kali.deb \
+  && rm kali.deb
```

Comment:
If you're going to split this, can you untab the first line so they're both indented the same?

Reply:
Heh, two tab characters snuck in somehow. Fixed.